response_module/response_controller.py

```
@@ -74,6 +97,26 @@ def send(self):
+
+        comment_msg_groups = {}
+        with self.comment_lock:
+            for msg in self.inline_comments:
+                if 'COMMENT' in msg['msg_type']:
+                    self.__parse_comment_msg(msg, comment_msg_groups)
+        self.inline_comments = []
+
+        ret = True
+        for target, msg_group in comment_msg_groups.items():
+            reply_target = ResponseFactory.get_message_instance(target, self.config)
+            for group, messages in msg_group.items():
```

Ruff (0.8.2)

Loop control variable `group` not used within loop body

Rename unused `group` to `_group`

(B007)